packages/plugins/rubygem-foreman-tasks/rubygem-foreman-tasks.spec

```diff
@@ -47,43 +47,22 @@ Provides: foreman-plugin-%{plugin_name} = %{version}
 # end specfile generated dependencies
 
 # start package.json devDependencies BuildRequires
-BuildRequires: %{?scl_prefix}npm(babel-cli) >= 6.10.1
-BuildRequires: %{?scl_prefix}npm(babel-cli) < 7.0.0
-BuildRequires: %{?scl_prefix}npm(babel-core) >= 6.26.3
-BuildRequires: %{?scl_prefix}npm(babel-core) < 7.0.0
-BuildRequires: %{?scl_prefix}npm(babel-loader) >= 7.1.1
-BuildRequires: %{?scl_prefix}npm(babel-loader) < 8.0.0
-BuildRequires: %{?scl_prefix}npm(babel-plugin-module-resolver) >= 3.2.0
-BuildRequires: %{?scl_prefix}npm(babel-plugin-module-resolver) < 4.0.0
-BuildRequires: %{?scl_prefix}npm(babel-plugin-syntax-dynamic-import) >= 6.18.0
-BuildRequires: %{?scl_prefix}npm(babel-plugin-syntax-dynamic-import) < 7.0.0
-BuildRequires: %{?scl_prefix}npm(babel-plugin-transform-class-properties) >= 6.24.1
-BuildRequires: %{?scl_prefix}npm(babel-plugin-transform-class-properties) < 7.0.0
-BuildRequires: %{?scl_prefix}npm(babel-plugin-transform-object-assign) >= 6.8.0
-BuildRequires: %{?scl_prefix}npm(babel-plugin-transform-object-assign) < 7.0.0
-BuildRequires: %{?scl_prefix}npm(babel-plugin-transform-object-rest-spread) >= 6.8.0
-BuildRequires: %{?scl_prefix}npm(babel-plugin-transform-object-rest-spread) < 7.0.0
-BuildRequires: %{?scl_prefix}npm(babel-preset-env) >= 1.7.0
-BuildRequires: %{?scl_prefix}npm(babel-preset-env) < 2.0.0
-BuildRequires: %{?scl_prefix}npm(babel-preset-react) >= 6.5.0
-BuildRequires: %{?scl_prefix}npm(babel-preset-react) < 7.0.0
+BuildRequires: %{?scl_prefix}npm(@babel/core) >= 7.7.0
+BuildRequires: %{?scl_prefix}npm(@babel/core) < 8.0.0
+BuildRequires: %{?scl_prefix}npm(@theforeman/builder) >= 4.0.2
+BuildRequires: %{?scl_prefix}npm(@theforeman/builder) < 5.0.0
+BuildRequires: %{?scl_prefix}npm(@theforeman/eslint-plugin-foreman) = 4.0.5
+BuildRequires: %{?scl_prefix}npm(@theforeman/stories) >= 4.0.2
+BuildRequires: %{?scl_prefix}npm(@theforeman/stories) < 5.0.0
+BuildRequires: %{?scl_prefix}npm(@theforeman/test) >= 4.0.2
+BuildRequires: %{?scl_prefix}npm(@theforeman/test) < 5.0.0
```

Comment:
Pretty sure these shouldn't be required

Reply:
Nice catch @tbrisker, the only `@theforeman/---` packages to use in packaging are `vendor` and `builder`.

Reply:
They'res blacklisted since 91e9e00b9a21c8379076c54bfb8a6eb4c0625576 and 665573f0b64535dc827eddeeaa3cad17eb5f2212 so maybe this was generated on an old version?